solr/server/solr/solr.xml

```diff
@@ -53,7 +54,6 @@
     class="HttpShardHandlerFactory">
     <int name="socketTimeout">${socketTimeout:600000}</int>
     <int name="connTimeout">${connTimeout:60000}</int>
-    <str name="shardsWhitelist">${solr.shardsWhitelist:}</str>
```

Comment:
If a user has a legacy config with this defined, what do we do now?  Log a warning would be nice.

Reply:
Yes, I added a warning in HttpShardHandlerFactory#init() line 228. Is it what you thought of?

Reply:
Yes; thanks.